js/index.js

```diff
@@ -0,0 +1,90 @@
+import("../pkg/index.js")
```

Comment:
index.js is actually unchanged, except that now everything happens in a closure, which execute once wasm is loaded. Surley there is a more elegant way, I just don't know it yet.